### ISSUE TITLE
loader.py refers to undefined variables "e"

### ISSUE THREAD

**Issue by leisti** [codexns/sublime-ssl-linux#3]
I took a look at the console output after opening Sublime Text, and noticed this:

```
Reloading plugin /root/.config/sublime-text-2/Packages/0_package_control_loader/01-ssl-linux.py
Traceback (most recent call last):
  File "./sublime_plugin.py", line 62, in reload_plugin
  File "./01-ssl-linux.py", line 26, in <module>
    console_write(u'Linux SSL: _ssl module import error - %s' % unicode_from_os(e))
NameError: name 'e' is not defined
```

I googled for the offending code line, and got exactly one hit, on this Github project, file
sublime-ssl-linux/loader.py.

``` python
except (ImportError):
  console_write(u'Linux SSL: _ssl module import error - %s' % unicode_from_os(e))
  sys_path.remove(lib_path)
  continue
```

I haven't studied python, but it looks like the variable e needs to be defined on the "except" line,
twice in that file.  Perhaps better check other exception-handling code, too?

**Comment by leisti** [comment]
That was in Sublime Text 2. I upgraded to the most recent beta of version 3, which just automatically downloaded an SSL library because, according to the information dialog, it isn't automatically included in ST3. So, the problem with the missing SSL module that revealed the output bug I reported doesn't occur anymore -- but the bug might still exist in the loader.py file.

**Comment by wbond** [comment]
This was fixed with cbb5d384bfe9001dd0610191453706f3799624df

**Comment by ghost** [comment]
OK, thanks.
